modules/machineset-aws-enabling-gpu-support.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-aws-enabling-gpu-support_{context}"]
+= Enabling GPU support for a machine set
```

Comment:
Don't see this in the preview, and it also looks like it's GCP text from the similar GCP content